pkg/apis/pipeline/v1/pipelinerun_types.go

```diff
@@ -470,6 +470,8 @@ const (
 	// FinallyTimedOutSkip means the task was skipped because the PipelineRun has passed its Timeouts.Finally.
 	FinallyTimedOutSkip SkippingReason = "PipelineRun Finally timeout has been reached"
 	// None means the task was not skipped
+	MatrixParamsContainEmptyArrSkip SkippingReason = "PipelineRun tasks were skipped because matrix parameters contain empty arr"
```

Comment:
at first I thought there is a typo in `Arr` in `MatrixParamsContainEmptyArrSkip` 🙃 

NIT: Suggest to keep it simple, `EmptyArrayInMatrixParams` and set it to `"PipelineRun tasks were skipped because matrix parameters contain empty array"`

Reply:
Updated naming